cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorMockTest.java

```diff
@@ -349,6 +353,47 @@ public void testReconcile(Params params, VertxTestContext context) {
             .compose(v -> operator.reconcile(new Reconciliation("test-trigger", Kafka.RESOURCE_KIND, NAMESPACE, CLUSTER_NAME)))
             .onComplete(context.succeeding(v -> async.flag()));
     }
+
+    @ParameterizedTest
+    @MethodSource("data")
+    public void testPauseReconcile(Params params, VertxTestContext context) {
```

Comment:
The parametrized tests usually run many times with different configurations. I'm not sure this is really needed in here. Wouldn't it be better to put this into a different class without parametrization and run it just once?